mne/epochs.py

```diff
@@ -1233,6 +1242,16 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
+                if any([k_e not in list(epochs.event_id.keys())
+                       for k_e in key]):
+                    key = [k for k in list(epochs.event_id.keys())
+                           if all([set(k_i.split('/')).issubset(k.split("/"))
+                                   for k_i in key])]
+                    if len(key) == 0:
```

Comment:
My rationale: once you're inside _key_match, you don't know anymore if multi-indexing was even *attempted*.